### ISSUE TITLE
`scoped` keyword is not displayed in tooltip

### ISSUE THREAD

**Issue by SunnieShine** [dotnet/roslyn#64439]
**Version Used**: 

VS2022 17.4p2.1

**Steps to Reproduce**:

Type code like:

```csharp
static void Method(scoped ref Span<char> instance)
{
    // ...
}
```

The tooltip doesn't show `scoped` keyword.

![image](https://user-images.githubusercontent.com/23616315/193597655-4a161084-d6d1-4b07-a8fb-f2bd1895ff03.png)

**Expected Behavior**:

`scoped` keyword should be correctly displayed.

**Actual Behavior**:

No `scoped` keyword displayed.

**Comment by Youssef1313** [comment]
Possibly fixed in #64285 ?

**Comment by SunnieShine** [comment]
@Youssef1313

Sorry for dupe... I'll close it.

Too fast to response 🤣
